src/WriteTask.spec.ts

```diff
@@ -9,7 +9,7 @@ describe("WriteTask", () => {
 
   async function assertRoundTrip(args: {
     tag: keyof WriteTags
-    inputValue: string | number | string[]
+    inputValue: string | number | string[] | { [key: string]: string }[]
```

Comment:
Can't the values be numeric as well?
```suggestion
    inputValue: string | number | string[] | { [key: string]: string | number }[]
```